ftw/mobilenavigation/browser/resources/navigation.css

```diff
@@ -41,7 +41,7 @@ div.slideNavi.loading,
 #portal-globalnav.mobileNavigation {
```

Comment:
Yep, but I was just being consistent with the rest of your product.  19 other rules in this CSS file also use imply #portal-globalnav.mobileNavigation.  I can take it out in my rule, would you care to remove it from the others?